### ISSUE TITLE
H36m test set for Hybrik

### ISSUE THREAD

**Issue by no-Seaweed** [open-mmlab/mmhuman3d#271]
Hi, I have been searching around 'Sample_5_train_Human36M_smpl_leaf_twist_protocol_2.json' and the test-with-sample-20 one for a while. May I kindly ask where can I get these files? Currently, I have already have h36m_mosh_valid_p2.npz, but not the original mosh file. Do you have a file converting this to Hybrik version?

Best,
S.F.

**Comment by ttxskk** [comment]
Hi @no-Seaweed,

You can find the JSON file [here](https://github.com/open-mmlab/mmhuman3d/blob/main/docs/preprocess_dataset.md#hybrik), and then you can refer to [convert_datasets](https://github.com/open-mmlab/mmhuman3d/blob/main/docs/preprocess_dataset.md#datasets-for-supported-algorithms) to generate the training data.

**Comment by LyThuyPhuong** [comment]
I run demo for HyrIK but it has confliction between openpifpaf and pytorch3d, can you please show me the script to install these libraries and demo script for HyrIK. Thank you so much!

**Comment by ttxskk** [comment]
Hi @LyThuyPhuong, 

Sorry, we haven't supported the inference of HybrIK.

**Comment by LyThuyPhuong** [comment]
Excuse me, can I run pymafx_estimate_smplx.py for HyrIK demo?

**Comment by LyThuyPhuong** [comment]
Excuse me, can I asked which part that you're missing to demo HyrIK?

**Comment by ttxskk** [comment]
Hi @LyThuyPhuong, 

You need to use RootNet to get joint roots, which is not supported in MMHuman3D. I suggest you refer to the official codebase because the author has updated a new version that can be used directly for the demo.

Thanks

**Comment by ttxskk** [comment]
The official codebase of [Hybrik](https://github.com/Jeff-sjtu/HybrIK). @LyThuyPhuong

**Comment by LyThuyPhuong** [comment]
Thank you for this response.

**Comment by no-Seaweed** [comment]
> Hi @no-Seaweed,
> 
> You can find the JSON file [here](https://github.com/open-mmlab/mmhuman3d/blob/main/docs/preprocess_dataset.md#hybrik), and then you can refer to [convert_datasets](https://github.com/open-mmlab/mmhuman3d/blob/main/docs/preprocess_dataset.md#datasets-for-supported-algorithms) to generate the training data.

Thank you very much. Closing the issue.
